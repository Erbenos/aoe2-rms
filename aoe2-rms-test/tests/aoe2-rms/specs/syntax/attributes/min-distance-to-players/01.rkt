#lang aoe2-rms
; Place the starting scout at a distance of min 7 tiles.

<OBJECTS-GENERATION>
(create-object 'SCOUT
               (set-place-for-every-player)
               (min-distance-to-players 7))
