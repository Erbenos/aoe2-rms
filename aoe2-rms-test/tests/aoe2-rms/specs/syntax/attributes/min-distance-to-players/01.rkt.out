<OBJECTS_GENERATION>
create_object SCOUT {
  set_place_for_every_player
  min_distance_to_players 7
}
